molgenis-api-tests/src/test/java/org/molgenis/api/tests/rest/v1/RestControllerV1APIIT.java

```diff
@@ -591,6 +592,31 @@ public void testUpdateAttributePut() {
         .statusCode(OKE);
   }
 
+  @Test
+  public void testUpdateAttributePutWithEmptyValue() {
+    given()
+        .log()
+        .all()
+        .contentType(APPLICATION_JSON)
+        .header(X_MOLGENIS_TOKEN, testUserToken)
+        .put(API_V1 + "V1_API_TypeTestAPIV1/1/xstringnillable")
+        .then()
+        .log()
+        .all()
+        .statusCode(OKE);
+
+    given()
+        .log()
+        .all()
+        .header(X_MOLGENIS_TOKEN, testUserToken)
+        .get(API_V1 + "V1_API_TypeTestAPIV1/1/xstringnillable")
+        .then()
+        .log()
+        .all()
+        .statusCode(OKE)
+        .body("xstringnillable", nullValue());
```

Comment:
should be before the then I think?

Reply:
?? no ? send out the 'get' to check if update was succes, then when the get returns check the result